mackup/applications/docker.cfg

```diff
@@ -3,3 +3,4 @@ name = Docker
 
 [configuration_files]
 .dockercfg
+.docker
```

Comment:
Seems like syncing the entire `.docker` folder is excessive as this includes all the docker machines as well, which can each be massive (over 30GB for me)